hw/dv/sv/cip_lib/doc/index.md

```diff
@@ -71,6 +71,17 @@ The following is a list of common features and settings:
 * **ral**: This is the instance to the auto-generated RAL model that is
   extended from `dv_base_reg_block`. In the base class, this is created using
   the RAL_T class parameter which the extended IP env cfg class sets correctly.
+* **tl_intg_alert_name**: This the name of the alert that will be triggered once
+  a TLUL integrity error occurs. If the block isn't used the default alert name
+  `fatal_fault`, the user should override it at `initialize`
```

Comment:
maybe follow the previous bullet point and add a period? same for line 77

Reply:
Good point. Updated, thanks